modules/update-upgrading-web.adoc

```diff
@@ -47,9 +47,9 @@ such as `stable-{product-version}`.
 ====
 For production clusters, you must subscribe to a stable-* or fast-* channel.
 ====
-** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *Update Status* is not *Updates Available*, you cannot upgrade your
 cluster.
-** The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** *Select Channel* indicates the cluster version that your cluster is running
```

Comment:
Here should be:
”Last Completed Version“ indicates the cluster version that your cluster is running.